resdk/resources/data.py

```diff
@@ -201,6 +199,20 @@ def descriptor_schema(self, dschema):
         # Save descriptor schema if already hydrated, otherwise it will be rerived in getter
         self._hydrated_descriptor_schema = dschema if is_descriptor_schema(dschema) else None
 
+    @property
+    def started(self):
+        """Start time."""
+        if 'started' not in self._original_values:
+            raise ValueError('Instance does not have `started` attribute.')
```

Comment:
This is not needed.

Reply:
Why not? `resdk.resources.Data` can be created locally and no `started` value will exist in `_original_values`. Do we return `None` or do we raise an error? Same goes for all: `created`, `modified`, `started`, `finished`.

Reply:
I see. The usual approach that we use for such cases is `if not self.id: raise`.